mlflow/projects/__init__.py

```diff
@@ -422,13 +422,17 @@ def _create_run(uri, experiment_id, work_dir, entry_point):
         parent_run_id = existing_run.info.run_uuid
     else:
         parent_run_id = None
-    active_run = tracking.MlflowClient().create_run(
+    client = tracking.MlflowClient()
+    active_run = client.create_run(
         experiment_id=experiment_id,
         source_name=source_name,
         source_version=_get_git_commit(work_dir),
         entry_point_name=entry_point,
         source_type=SourceType.PROJECT,
         parent_run_id=parent_run_id)
+    if mode == "local" and databricks_utils.is_in_databricks_notebook():
```

Comment:
If in a notebook and we're executing a project locally (on the driver), set tags indicating which notebook we're in etc, so that we can tie the project run back to the notebook later on